image-loader/test/scala/lib/ImageProcessorLoaderTest.scala

```diff
@@ -143,21 +139,6 @@ class ImageProcessorLoaderTest extends FreeSpec with Matchers with EitherValues
       }
     }
 
-    "should fail to load multiple image processors if they don't meet the spec" in {
-      val conf:Configuration = Configuration.from(Map(
-        "some.path" -> List(
-          "scala.lib.NoArgImageProcessor",
-          Map(
-            "noClassName" -> "scala.lib.ConfigImageProcessor",
-            "config" -> Map("parameter" -> "value")
-          )
-        )
-      ))
-      val thrown = the[BadValue] thrownBy {
-        conf.get[Seq[ImageProcessor]]("some.path")
-      }
-      thrown.getMessage should include ("An image processor can either a class name (string) or object with className (string) and config (object) fields. This OBJECT is not valid.")
-    }
```

Comment:
Likewise - this is a useful test as it helps ensure that an image processor cannot easily be mis-configured.